backends-common/opensearch/src/main/java/org/apache/james/backends/opensearch/OpenSearchIndexer.java

```diff
@@ -90,7 +89,7 @@ public Mono<BulkResponse> update(List<UpdatedRepresentation> updatedDocumentPart
             op -> op.update(idx -> idx
                 .index(aliasName.getValue())
                 .id(updatedDocumentPart.getId().asString())
-                .document(Collections.singletonMap("doc", new RawValue(updatedDocumentPart.getUpdatedDocumentPart())))
+                .document(new RawValue(updatedDocumentPart.getUpdatedDocumentPart()))
```

Comment:
Actually they seem to have fixed that dirty workaround, we can use a json raw value directly in the document update again :)